charts/victoria-logs-single/values.yaml

```diff
@@ -150,35 +150,35 @@ server:
     #   cpu: 500m
     #   memory: 512Mi
 
-  # Indicates whether the Container is ready to service requests. If the readiness probe fails, the endpoints controller removes the Pod's IP address from the endpoints of all Services that match the Pod. The default state of readiness before the initial delay is Failure. If a Container does not provide a readiness probe, the default state is Success.
-  readinessProbe:
-    httpGet:
-      path: /health
-      port: http
-    initialDelaySeconds: 5
-    periodSeconds: 15
-    timeoutSeconds: 5
-    failureThreshold: 3
-
-  # Indicates whether the Container is running. If the liveness probe fails, the kubelet kills the Container, and the Container is subjected to its restart policy. If a Container does not provide a liveness probe, the default state is Success.
-  livenessProbe:
-    httpGet:
-      scheme: HTTP
-      path: /health
-      port: 9428
-    initialDelaySeconds: 30
-    periodSeconds: 30
-    timeoutSeconds: 5
-    failureThreshold: 10
-
-  # Indicates whether the Container is done with potentially costly initialization. If set it is executed first. If it fails Container is restarted. If it succeeds liveness and readiness probes takes over.
-  startupProbe: {}
-    #tcpSocket:
-    #  port: http
-    #failureThreshold: 30
-    #periodSeconds: 15
-    #successThreshold: 1
-    #timeoutSeconds: 5
+  probe:
+    # Indicates whether the Container is ready to service requests. If the readiness probe fails, the endpoints controller removes the Pod's IP address from the endpoints of all Services that match the Pod. The default state of readiness before the initial delay is Failure. If a Container does not provide a readiness probe, the default state is Success.
+    readiness:
+      httpGet:
+        path: '{{ include "vm.probe.http.path" . }}'
+        scheme: '{{ include "vm.probe.http.scheme" . }}'
+        port: '{{ include "vm.probe.port" . }}'
```

Comment:
The tpl doesn't seem to work in my test:
```
➜  victoria-logs-single git:(vmcluster-override-probes) ✗ helm template .
Error: template: victoria-logs-single/templates/server-statefulset.yaml:61:30: executing "victoria-logs-single/templates/server-statefulset.yaml" at <include "vm.probe" (dict "app" $app "type" "readiness")>: error calling include: template: victoria-logs-single/templates/_helpers.tpl:155:4: executing "vm.probe" at <tpl (toYaml $probe) .>: error calling tpl: cannot retrieve Template.Basepath from values inside tpl function: failureThreshold: 3
httpGet:
  path: '{{ include "vm.probe.http.path" . }}'
  port: '{{ include "vm.probe.port" . }}'
  scheme: '{{ include "vm.probe.http.scheme" . }}'
initialDelaySeconds: 5
periodSeconds: 15
timeoutSeconds: 5: "BasePath" is not a value

Use --debug flag to render out invalid YAML
```

Reply:
are you trying to run it from my branch?

Reply:
yeah, I pulled the latest

Reply:
could you please upgrade helm locally?

Reply:
there were issues with old versions on CI as well

Reply:
Hmm, upgrading from helm v3.12.3 to v3.15.3 solves the error, should related to https://github.com/helm/helm/issues/11168#issuecomment-1935567803.
We might need to clarify the helm version somewhere.

Reply:
added to requirements.md